Gruntfile.js

```diff
@@ -376,39 +383,26 @@ module.exports = function (grunt) {
 
     // compile the single JSX file into the appropriate Fauxton folder
     var targetFilepath = filepath;
-    if (isJSX) {
-      var folder = filepath.replace(/\/[^\/]*$/, '');
-      var targetFolder = folder;
-
-      // if the JSX file ISN'T in Fauxton, generate the .js file there
-      if (!(/^app\/addons/.test(folder))) {
-        targetFolder = folder.replace(/.*\/addons/, 'app/addons');
-      }
-      targetFilepath = filepath.replace(/.*\/addons/, 'app/addons');
-      targetFilepath = targetFilepath.replace(/\.jsx$/, '.js');
 
-      config.shell['build-single-jsx'].command = 'node ./node_modules/react-tools/bin/jsx -x jsx ' + folder + ' ' + targetFolder + ' --no-cache-dir';
-      grunt.task.run(['shell:build-single-jsx']);
-    }
+    var folder = filepath.replace(/\/[^\/]*$/, '');
+    var targetFolder = folder;
 
-    // if the JS file that just changed was outside of Fauxton, copy it over
-    if (isJS && !(/^app\/addons/.test(filepath))) {
-      config.copy.changedFiles.files = [{
-        src: filepath,
-        dest: filepath.replace(/.*\/addons/, 'app/addons')
-      }];
-      grunt.task.run(['copy:changedFiles']);
-    }
```

Comment:
i am not sure about the intention of the code code and it does not seem to make sense with all required js and compiled js files in the new dev server directory any more - can you confirm that @benkeen ?